CS342MemberViews/Sources/JenMoore/JenMoore.swift

```diff
@@ -0,0 +1,53 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+public struct JenMoore: View {
+    @Binding var navigationPath: NavigationPath
```

Comment:
If you do not use the `navigationPath` you can remove the property here:
```suggestion
```

And then also adjust the initializer to have no argument:
```swift
    public init() { }
```